### ISSUE TITLE
Wazuh cluster causes work node alarm, disk IO and CPU alarm. I don't know how to solve it

### ISSUE THREAD

**Issue by huoshenyingmu** [wazuh/wazuh#4931]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.11.4 | Wazuh component | Manager | Packages | centos 7 |

1、After I installed the wazuh cluster correctly, there was a problem
```
#manager Run the command to view the cluster status
time /var/ossec/bin/cluster_control -l
NAME            TYPE    VERSION  ADDRESS         
manager         master  3.11.4   192.168.0.1
wazuh-worker-2  worker  3.11.4   192.168.0.2
wazuh-worker-1  worker  3.11.4   192.168.0.3

real    0m0.160s
user    0m0.134s
sys     0m0.026s
```
```
#worker-1 Run the command to view the cluster status
/wazuh-data/ossec/bin/cluster_control -l
ERROR: Error 3009 - Error executing distributed API request: Error sending request: timeout expired.
#After several more runs, you can get the correct result
NAME            TYPE    VERSION  ADDRESS         
manager         master  3.11.4   192.168.0.1  
wazuh-worker-2  worker  3.11.4   192.168.0.2
wazuh-worker-1  worker  3.11.4   192.168.0.3

real    0m13.269s
user    0m0.173s
sys     0m0.038s
```
1.1、Current CPU and other resource usage of work nodes
![image](https://user-images.githubusercontent.com/27325498/79710674-505d1800-82f8-11ea-9ee6-aab7da11b0e6.png)
![image](https://user-images.githubusercontent.com/27325498/79710698-61a62480-82f8-11ea-8468-70ece437da6c.png)

|Wazuh version|OS|Kernel number|Memory|disk|Platform|
|---|---|---|---|---|--|
| 3.11.4 | centos 7 | 2c | 4G | 500G | aws ec2 |

It's been a long time

**Comment by huoshenyingmu** [comment]
![image](https://user-images.githubusercontent.com/27325498/79710823-b8136300-82f8-11ea-9f5a-78efdefc5a49.png)
This is the configuration of the work node

**Comment by AdriiiPRodri** [comment]
Hi huoshenyingmu,

We have been investigating this issue and have come to a few conclusions. This may be happening for two main reasons:

- It is possible that there are **network problems**. When cluster_control binary is run in the master, it doesn't have to make any request. It just returns the information it has. Therefore, this would explain why in the master node the call works correctly and in the worker nodes it does not. By default, the cluster timeout is 20 seconds. There were several memory leaks in the cluster which have been fixed recently. The most important one was corrected in `3.11.2`, but this should not be the problem. A reading of 162 MB/s can be possible in the synchronization periods. These periods are very fast and **the peak is temporal**.

- Secondly, the cause we consider most probable is the **lack of EBS credits** in the worker's machine ([EC2 instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html)). When one of the machines runs out of credits, it does not disconnect but its computing capacity is very reduced, causing a failure in the reading and writing of the synchronization process of the cluster. This would explain the processor's percentage in iowait state. When the machine gets credits again, everything should work correctly while the credits are not over. Is the machine where the master is installed the same as the workers? If so, what type of AWS instance is it? ([EC2 pricing](https://aws.amazon.com/ec2/pricing/on-demand/?nc1=h_ls))

These are the possible causes that we have considered. Please share with us the master's `cluster.log` file (`/var/ossec/logs/cluster.log`) and the `cluster.log` file (`/wazuh-data/ossec/logs/cluster.log`). This way we can verify if everything is correct or if the cluster has a strange behavior.

Best regards,
Adrián Peña

**Comment by vikman90** [comment]
Closing this issue due to inactivity.
